extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -140,8 +140,64 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		await this.bookTocManager.createBook(bookPath, contentPath);
 	}
 
-	async editBook(book: BookTreeItem, section: BookTreeItem): Promise<void> {
-		await this.bookTocManager.updateBook(section, book);
+	async editBook(movingElement: BookTreeItem): Promise<void> {
+		try {
+			let bookOptions: vscode.QuickPickItem[] = [];
+			let pickedSection: vscode.QuickPickItem;
```

Comment:
Looks like only bookOptions is used throughout the entire method. These other variables could be declared inline wherever they get used.